model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala

```diff
@@ -0,0 +1,54 @@
+package hmda.model.census
+
+import java.io.File
+
+import scala.io.Source
+
+object CBSAMetroMicroLookup {
+  val values: Seq[CBSAMetroMicro] = {
+    val lines = Source.fromFile(new File("model/src/main/resources/tl_2013_us_cbsa.csv")).getLines
+    lines.map { line =>
+      val values = line.split('|').map(_.trim)
+      val CSAFP = values(0)
```

Comment:
This val is unused. I may have more to say about that based on the outcome of other parts, but I'm noting it now so we don't forget.